### PR TITLE
OSDOCS#5683: [4.12] Update the `osdk_ver_n1` attribute to 1.22.2

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.12

Issue: [OSDOCS-5683](https://issues.redhat.com//browse/OSDOCS-5683)

Link to docs preview:

* [Updating Go-based Operators](https://58289--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/golang/osdk-golang-updating-projects.html)
* [Updating Ansible-based Operators](https://58289--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/ansible/osdk-ansible-updating-projects.html)
* [Updating Helm-based Operators](https://58289--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/helm/osdk-helm-updating-projects.html)
* [Updating Hybrid Helm-based Operators](https://58289--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/helm/osdk-hybrid-helm-updating-projects.html)
* [Updating Java-based Operators](https://58289--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/java/osdk-java-updating-projects.html)
* [Notable technical changes > Operator SDK](https://58289--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-notable-technical-changes) (scroll to Operator SDK update)

_Places where hard-coded OSDK 1.25.4 version was replaced with `osdk_ver` attribute_
* [Notable technical changes > Operator SDK](https://58289--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-notable-technical-changes)
* [About Operator SDK](https://58289--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/osdk-about.html)

QE review:
- [x] QE has approved this change.

Additional information: The Operator SDK version was updated to 1.22.2 in OCP 4.11.34.

`enterprise-4.11` PR: #58106

### CODE REVIEW

File: _attributes/common-attributes.adoc [hunk 1]
```diff
@@ -170,4 +170,4 @@ endif::[]
 //Operator SDK version
 :osdk_ver: 1.25.4
 //Operator SDK version that shipped with the previous OCP 4.x release
-:osdk_ver_n1: 1.22.0
+:osdk_ver_n1: 1.22.2
```
Comment (everettraven) [inline _attributes/common-attributes.adoc:173]: 4.12 ships with osdk 1.25.4 currently: https://github.com/openshift/ocp-release-operator-sdk/blob/e11bcad257328713bfaa1f42160dc4d8e396cfa7/patches/03-setversion.patch#L10

```suggestion
:osdk_ver_n1: 1.25.4
```

**Review by everettraven — CHANGES_REQUESTED** [review]

File: _attributes/common-attributes.adoc (hunk 1 above)
Comment (michaelryanpeter) [reply _attributes/common-attributes.adoc:173]: @everettraven: The attribute `osdk_ver_n1` is the version of the SDK that ships with the previous release, in this case 4.11. We use this attribute a couple of times in the upgrade docs. For example, the following markup in the [release notes:](https://58289--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-operator-sdk-1-25-z)
`If you have Operator projects that were previously created or maintained with Operator SDK {osdk_ver_n1}, update your projects to keep compatibility with Operator SDK {osdk_ver}.`

Renders as:
If you have Operator projects that were previously created or maintained with Operator SDK 1.22.2, update your projects to keep compatibility with Operator SDK 1.25.4.

Comment (everettraven) [reply _attributes/common-attributes.adoc:173]: Ahhhh gotcha. That makes sense - thanks for the clarification!

**Review by everettraven — APPROVED** [review]

File: _attributes/common-attributes.adoc (hunk 1 above)
Comment (everettraven) [reply _attributes/common-attributes.adoc:173]: /resolve

**Comment by xltian** [comment]
ack

**Comment by oceanc80** [comment]
LGTM

**Review by sheriff-rh — APPROVED** [review]
Looks great! :rocket:

**Pull request merged**